elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -94,6 +94,8 @@ export enum MultisignatureStatus {
 	FAIL = 4,
 }
 
+const typeMultiSignature = 4;
```

Comment:
```suggestion
const MULTISIGNATURE_TYPE = 4;
```

Reply:
Would you add the suggestion to the usage of this cont so I can commit it working without an additional commit?

Reply:
Renamed